CDCgov/prime-reportstream
# 11512 - About landing page: Update Recommended resources card descriptions

This PR ...

About landing page: Update Recommended resources card descriptions

Test Steps:

1. Go to 'reportstream.cdc.gov/about'
2. Scroll down to 'Recommended resources'
3. See the card descriptions

## Changes
 
![Screenshot 2023-10-02 at 6 45 58 PM](https://github.com/CDCgov/prime-reportstream/assets/102491809/0ab7c0fb-3c08-4980-8082-71dcbff62d78)


## Checklist

### Testing

## Linked Issues
- Fixes #11512
